app/src/main/java/com/sun_asterisk/moviedb_49/data/source/MovieDataSource.kt

```diff
@@ -0,0 +1,18 @@
+package com.sun_asterisk.moviedb_49.data.source
+
+import com.sun_asterisk.moviedb_49.data.model.Movie
+import com.sun_asterisk.moviedb_49.data.source.remote.OnFetchDataJsonListener
+
+class MovieDataSource private constructor(private val remote: MovieDBDataSource.Remote) {
```

Comment:
```suggestion
class MovieRepository private constructor(private val remote: MovieDBDataSource.Remote) {
```
tạo package repository rồi bỏ vào đó.